assets/src/blocks/Socialshare/SocailshareBlock.js

```diff
@@ -0,0 +1,273 @@
+import { Socialshare } from './Socialshare';
```

Comment:
There is a typo on the name of this file. 

Also can you fix indentation and linting errors?

`./node_modules/eslint/bin/eslint.js assets/src/blocks/Socialshare/SocailshareBlock.js`